dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/FATSuite.java

```diff
@@ -29,7 +32,21 @@
                 ConnectionManagerMBeanTest.class
 })
 public class FATSuite {
+
+    public static final String javaeeServer = "com.ibm.ws.jca.fat";
+    public static final String jakartaeeServer = "com.ibm.ws.jca.fat.jakarta";
+
     @ClassRule
     public static RepeatTests r = RepeatTests.withoutModification()
-                    .andWith(FeatureReplacementAction.EE8_FEATURES());
-}
\ No newline at end of file
+                    .andWith(FeatureReplacementAction.EE8_FEATURES())
+                    .andWith(new JakartaEE9Action());
+
+    public static LibertyServer getServer() {
+        if (JakartaEE9Action.isActive()) {
+            return LibertyServerFactory.getLibertyServer(jakartaeeServer);
+        } else {
+            return LibertyServerFactory.getLibertyServer(javaeeServer);
```

Comment:
Normally the feature replacement repeat action would update the server for you. Is there a reason that doesn't work in this case?

Reply:
The feature replacement action just replaces features, but in this case the server.xml file has configurations like: 
```
    <jmsActivationSpec id="FVTMessageDrivenBeanBindingOverride">
      <properties.FAT1 destinationRef="topic1" destinationType="javax.jms.Topic"/>
    </jmsActivationSpec>
```

Instead of trying to run the transformer on the server.xml file, I figured it would be easier and quicker just to make a copy and change the configuration manually.

Reply:
That's a good point.  I'm sure making the copy is easier for now, but it will be more difficult in the long term to maintain.  With two copies, we will inevitably end up with developers making a fix or improvement in only one of the files and they will get out of sync.  It seems like it ought to be something the EE9 replacement action can do to update config element values automatically with the simplicity config support in place.  Does that not apply here because this is generated metatype/config?  Otherwise it ought to be possible.

Reply:
Opened this as a new issue here: https://github.com/OpenLiberty/open-liberty/issues/14374